build.sbt

```diff
@@ -1,5 +1,6 @@
 // Copyright © 2017-2020 UKG Inc. <https://www.ukg.com>
 
+import Dependencies.autoImport.OpenTracing.jaeger
```

Comment:
surge docs now contains some example code that shows how to initialize a jaeger tracer so that's why we need this here